src/test/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultMonitorServiceTest.java

```diff
@@ -66,15 +67,15 @@
   private Future<?> task;
   @Mock
   private HostInfo info;
-  @Mock
-  private PropertySet set;
 
+  private PropertySet set;
   private AutoCloseable closeable;
   private DefaultMonitorService monitorService;
   private ArgumentCaptor<MonitorConnectionContext> contextCaptor;
 
   @BeforeEach
   void init() {
+    set = new DefaultPropertySet();
```

Comment:
set is too vague for a variable name, what about?

propertySet = new DefaultPropertySet();